sdk/python/kfp/containers/_component_builder.py

```diff
@@ -275,7 +275,8 @@ def build_docker_image(staging_gcs_path, target_image, dockerfile_path, timeout=
     target_image (str): gcr path to push the final image
     dockerfile_path (str): local path to the dockerfile
     timeout (int): the timeout for the image build(in secs), default is 600 seconds
-    namespace (str): the namespace within which to run the kubernetes kaniko job, default is "kubeflow"
+    namespace (str): the namespace within which to run the kubernetes kaniko job. Default is None, build_python_component 
```

Comment:
Could you copy the comments to the build_python_component func as well? 
And, adjust the words here "build_python_component"

Reply:
done